sites/cnfpt/src/backend/cnfpt/settings.py

```diff
@@ -192,20 +205,49 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     SOCIAL_AUTH_EDX_OAUTH2_ENDPOINT = values.Value()
     SOCIAL_AUTH_POSTGRES_JSONFIELD = False
 
+    # AUTHENTICATION
```

Comment:
Should be under "# LMS" as this is not really authentication for the richie app...